### PR TITLE
AB Micro: publish with filters

### PR DESCRIPTION
**Pull request by annagav**
#### What are the relevant tickets?

Depends on #710 
#### What's this PR do?

Publish an assignment with microscopy filtered images.

### CODE REVIEW

**Comment by noisecapella** [comment]
What should I notice for this PR?

File: instructor/compiler.py [hunk 1]
```diff
@@ -303,27 +303,56 @@ def facs_kinds(assignment):
...
+    filters = ['red', 'blue', 'green', 'merge']
     for sample_prep in a.microscopy_sample_prep.all():
         for mapping in sample_prep.image_mapping.all():
             key = "{sp.micro_analysis}%%{sp.condition}%%SP_ID_{protocol}".format(
                 sp=sample_prep,
                 protocol=mapping.strain_protocol.id
             )
-            if mapping.images.count() > 0:
+            if (
+                (
```
Comment (noisecapella) [inline instructor/compiler.py]: Can you move this comparison into a helper function since it's used below as well?

**Comment by annagav** [comment]
@noisecapella, In this PR you should able to preview the assignment you created with all the different images for filters.

**Comment by noisecapella** [comment]
:+1:

**Pull request merged**